lib/manager/conan/extract.ts

```diff
@@ -0,0 +1,89 @@
+import { ConanDatasource } from '../../datasource/conan';
+import { logger } from '../../logger';
+import { regEx } from '../../util/regex';
+import * as conan from '../../versioning/conan';
+import type { PackageDependency, PackageFile } from '../types';
+
+const regex = regEx(
+  `(?<name>[-_a-z0-9]+)/(?<version>[^@\n{*"']+)(?<userChannel>@[-_a-zA-Z0-9]+/[^\n.{*"' ]+)?`
+);
+
+function setDepType(content: string, originalType: string): string {
+  let depType = originalType;
+  if (content.includes('python_requires')) {
+    depType = 'python_requires';
+  } else if (content.includes('build_require')) {
+    depType = 'build_requires';
+  } else if (content.includes('requires')) {
+    depType = 'requires';
+  }
+  return depType;
+}
+
+export default function extractPackageFile(
```

Comment:
```suggestion
export function extractPackageFile(
```

Please no default exports